index.html

```diff
@@ -37,46 +37,192 @@
+          <a href="#" class="logo"><h3>Micky's</h3></a>
+          <nav>
+            <a href="#" id="hamburger"
+              ><i class="fa fa-bars" aria-hidden="true"></i
+            ></a>
+          </nav>
+        </div>
+        <ul class="menu_items">
+          <li class="close_button">
+            <img class="x_sign" src="./images/close_btn.svg" alt="Closing menu button" />
+          </li>
+          <li class="goto"><a href="#Portfolio">Portfolio</a><hr /></li>        
+          <li class="goto"><a href="#About">About</a><hr /></li>        
+          <li class="goto"><a href="#Contact">Contact</a><hr /></li>        
+        </ul>
+      </header>
+      <header class="desktop_toolbar">
+        <h3 class="logo">Micky's</h3>
+        <div class="desktop_options">
+          <h3>Portfolio</h3>
+          <h3>About</h3>
+          <h3>Contact</h3>
+          <h3><i class="fa fa-envelope" aria-hidden="true"></i></h3>
+        </div>
+      </header>
+      <section class="intro">
+        <div class="content">
+          <h1>Hey There.<br />I'm Michael</h1>
+          <h2>I’am a Software Developer</h2>
+          <p>
+            I can help you build a product , feature or website Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, don’t hesitate to contact me.
+          </p>
+        </div>
+        <div class="social_icons">
+          <ul class="socials">
+            <li>
+              <a href=""><i class="fab fa-github"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-linkedin-in"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-angellist"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-medium-m"></i></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="recent_works">
+        <div class="works_title">
+          <h3>My Recent Works</h3>
+          <hr />
+        </div>
+        <div class="headlines">
+          <div class="image_headline">
+            <img src="./images/Img_Placeholder.png" alt="Image place holder" />
+          </div>
+          <div class="discription_headline">
+            <h4>Multi-Post Stories</h4>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text ever
+              since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="tags">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Rubby</li>
+            </ul>
+            <button class="popup_display_button" id="1" type="button">See Project</button>
+          </div>
```

Comment:
# ♻ Required changes ♻

-  In all the `See Project` buttons I click and for some reason neither in the desktop version nor the mobile version that works, it doesn't redirect me to the pop-up window, it might be due to the changes that you recently have added to your project, please verify this feature on your deployed version, and test for it to work on your deployed version as well as locally, for it to be then ready to be approved, is just one step closer to get that approval you guys need, so hands-on to do a great job 💪😁♻Don't get discouraged you're almost there !!

### Recommendations :

I encourage you to implement these changes and submit for the next review once they've been implemented successfully 😁📌📌
![giphy](https://user-images.githubusercontent.com/54015740/151449079-789b341e-e5f5-4134-a5ad-638c65ebb71f.gif)

Reply:
Every thing works well I have tried to check and recheck it more than 3 times. I think you opened the main-branch deployment branch, or you have downloaded the main branch code. That may be the case.

- All the see project buttons are working.
- All the see live and see source works well.

If there is any chance you can re edit your decision towards my pull request it will be nice

Reply:
@Micky373  For the moment when I opened it, it wasn't working, could have been a server error in github deployment which sometimes happens, I would gladly do a re-edition for your approval, but you are required to submit the project again, and it will show in my dashboard or the next reviewer will approve, in any case when you feel your project is completed you only have to proceed to submit for a review, and you would get your approval, is not needed to direct contact the reviewer, when we submit a review is not definite, is just that we send it to another server so we don't have that request available anymore, and we need for you to resubmit for a review again, so you get that approval in the next review.